.readthedocs.yaml

```diff
@@ -0,0 +1,14 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
```

Comment:
very minor: For docs we could boost to 3.12 I think.

Reply:
Good idea, added!